src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1663,6 +1674,7 @@ void uct_dc_mlx5_iface_set_ep_failed(uct_dc_mlx5_iface_t *iface,
      * purged - they need to be rescheduled when DCI will be recovered after an
      * error */
 
+    uct_dc_mlx5_fc_restore_wnd(iface, ep, NULL);
```

Comment:
do we really want to do it when UCT_DC_MLX5_EP_FLAG_ERR_HANDLER_INVOKED is true?
or when the ep is equal to iface->tx.fc_ep ?

Reply:
it simplifies the code, because we should do it in case of `flush(CANCEL)` was done, but `UCT_DC_MLX5_EP_FLAG_FLUSH_CANCEL` isn't set here.
so, we can call it many times, but it wouldn't perform any actions.
but we can check for `fc_ep` first to not restore FC window in case of FC  EP - done